Samples/Samples/ViewModel/CalendarEventAddAttendeeViewModel.cs

```diff
@@ -49,12 +49,19 @@ public string EmailAddress
             }
         }
 
-        bool required;
+        public ObservableCollection<string> AttendeeTypes { get; } = new ObservableCollection<string>()
```

Comment:
This doesn't need to be an observablecollection since it's a static list

Reply:
altered in latest commit